app/controllers/grades_controller.rb

```diff
@@ -75,8 +75,8 @@ def view_my_scores
     @questions = {} # A hash containing all the questions in all the questionnaires used in this assignment
     questionnaires = @assignment.questionnaires
     retrieve_questions questionnaires
-    # @pscore has the newest versions of response for each response map, and only one for each response map (unless it is vary rubric by round)
-    @pscore = @participant.scores(@questions)
+    # @participant_score has the newest versions of response for each response map, and only one for each response map (unless it is vary rubric by round)
```

Comment:
I think rename `@pscore` to `@participant_score` is reasonable. But as far as I know, `@pscore` is also used in some other files. So it will be better you change all occurrences of `@pscore` to `@participant_score`, although it is out of your project requirement.